src/workspacer/ConfigContext.cs

```diff
@@ -45,6 +45,7 @@ public ConfigContext()
 
             _defaultLayouts = () => new ILayoutEngine[] {
                 new TallLayoutEngine(),
+                new GoldenLayoutEngine(),
```

Comment:
this layout engine looks great! but I don't think that it should necessarily be in the "default" set of layouts.

would you be cool with removing it from the default set? I think that this would make more sense as an optional engine. I'm hoping that I get spend some more time on documentation in the future, which would help expose / show off these engines (that aren't defaults).

Reply:
Of course!